robottelo/ui/rhai.py

```diff
@@ -0,0 +1,14 @@
+# -*- encoding: utf-8 -*-
+""" Implements methods for RHAI"""
+
+from robottelo.ui.base import Base
+from robottelo.ui.locators import locators
+
+
+class RHAI(Base):
+    def view_registered_systems(self):
+        """To view the number of registered systems"""
+        result = self.wait_until_element(
+            locators['insights.registered_systems']
+        ).text
+        return result
```

Comment:
Shouldn't we build a list from the systems? Processing python native data types is better than just a plain string.